onnxruntime/core/providers/webnn/webnn_execution_provider.cc

```diff
@@ -337,6 +344,9 @@ common::Status WebNNExecutionProvider::Compile(const std::vector<FusedNodeAndGra
     node_compute_funcs.push_back(compute_info);
   }
 
+  // Explictly release the WebNN builder to free memory.
```

Comment:
```suggestion
  // Explicitly release the WebNN builder to free memory.
```
Whenever you're in the area next time...